### PR TITLE
TabView: Emit detach status in PaneRemoved event.

### PR DESCRIPTION
**Pull request by fatihacet**

### CODE REVIEW

File: src/components/tabs/tabview.coffee [hunk 1]
```diff
@@ -158,7 +158,7 @@ module.exports = class KDTabView extends KDScrollView
       else if firstPane = @getPaneByIndex 0
         @showPane firstPane
 
-    @emit "PaneRemoved", { pane, handle }
+    @emit "PaneRemoved", { pane, handle, isDetached: shouldDetach }
     return { pane, handle }
 
   removePaneByName:(name)->
```
Comment (fatihacet) [inline src/components/tabs/tabview.coffee]: At first I thought, we shouldn't emit `PaneRemoved` when pane is detached. But I fear to break other things by not emitting an event. @sinan what do you think about it?

Comment (gokmen) [inline src/components/tabs/tabview.coffee]: this way is correct, LGTM.

Comment (fatihacet) [inline src/components/tabs/tabview.coffee]: Thanks @gokmen for review and comment. I talked with @sinan and figured out that if pane is detached there won't be `pane.parent` reference and I can use it without touching the framework code. Emitting `isDetached` in `PaneRemoved` event doesn't make sense much. I will let you know if it works.

Comment (gokmen) [inline src/components/tabs/tabview.coffee]: I agree, it would be better to not change this on framework level. Thanks.

**Pull request closed without merging**